allure-testng/src/main/java/io/qameta/allure/testng/config/AllureTestNgConfig.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright 2022 Qameta Software OÜ
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.qameta.allure.testng.config;
+
+import io.qameta.allure.util.PropertiesUtils;
+
+import java.util.Properties;
+
+import static java.lang.Boolean.parseBoolean;
+import static java.lang.System.setProperty;
+
+public class AllureTestNgConfig {
+
+    public static final String ALLURE_TESTNG_HIDE_DISABLED_TESTS = "allure.testng.hide.disabled.tests";
+    private static boolean hideDisabledTests;
```

Comment:
why this one is static?

Reply:
If we make it non-static, then **loadConfigProperties()** needs to be non-static.
Consequently, we might need to instantiate AllureTestNgConfig in AllureTestNg constructor or do similar thing - not sure if makes sense though:

line: 148 
_this.config = new AllureTestNgConfig.loadConfigProperties();_

What are your thoughts?

Reply:
AllureTestNgConfig should be a simple POJO that accept all the configuration properties in constructor. AllureTestNgConfig.loadConfigProperties then is a simple factory method that loads Allure properties and passes values to constructor.

Reply:
Cool, I think I managed to implement following your suggestion. Please, let me know if it makes sense